core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/PostProcessingSimulation.kt

```diff
@@ -341,8 +341,8 @@ private fun makeAllowanceRanges(
     val res = ArrayList<AllowanceRange>()
     for (point in fixedPoints) {
         val baseTime =
-            envelope.interpolateTotalTimeClamp(point.offset.distance.meters) -
-                envelope.interpolateTotalTimeClamp(transition)
+            envelope.interpolateDepartureFromClamp(point.offset.distance.meters) -
+                envelope.interpolateDepartureFromClamp(transition)
```

Comment:
There's no stop there so it doesn't matter, it's just about semantics

```suggestion
            envelope.interpolateDepartureFromClamp(point.offset.distance.meters) -
                envelope.interpolateArrivalAtClamp(transition)
```

Reply:
As discussed did the opposite: `ArrivalAt(end) - DepartureFrom(start)` in https://github.com/OpenRailAssociation/osrd/pull/7609/commits/b7f044ebd08664f02948f7aafb19982fee469f65